### ISSUE TITLE
In-Depth: Footer Nav

### ISSUE THREAD

**Issue by rossvanderlinde** [newamericafoundation/newamerica-cms#1114]
![screen shot 2016-08-11 at 3 23 40 pm](https://cloud.githubusercontent.com/assets/13524901/17601762/b463ed1c-5fd7-11e6-9edf-1c7bd0bf00f5.png)

For feedback

**Comment by kjacks** [comment]
@rossvanderlinde I feel like it's a bit wordy and kind of hidden as well

**Comment by rossvanderlinde** [comment]
Do you like it being centered? I'm reluctant to break up the simplicity of previous and next being 50% width each
